Update blueberries.css

HugoBDesigner opened the pull request:
Fixed .language-dropdown__dropdown__results__item rule, improved readability in some rules, added new rules.

Please review the "improved readability" rules specifically, as they might look different in your instance.

HugoBDesigner committed:
Update blueberries.css

Fixed .language-dropdown__dropdown__results__item rule, improved readability in some rules, added new rules.

HugoBDesigner committed:
Update blueberries.css

Added "moved account" rules.

brodokk commented on blueberries.css:
```diff
@@ -539,6 +548,13 @@
+    .compose-form__poll-wrapper select option {
+        background-color: hsl(var(--hue), 16%, 19%);
+    /*     color: hsl(var(--hue), 17%, 15%); */
+    
```
I think their is a `}` missing around here.

Be careful maybe more code can be missing too!

HugoBDesigner replied:
Fixed it, thanks for letting me know! I have also looked more carefully into all other rules for potential similar issues and haven't found any other.

brodokk commented:
The language dropdown seems nearly good:

Before:

![image](https://github.com/brodokk/Blueberries-Mastodon-UI-user-theme/assets/76159594/8f24e894-4751-499e-a988-bf0efb42ae22)

After (the `e` is still in the search bar but not visible):

![image](https://github.com/brodokk/Blueberries-Mastodon-UI-user-theme/assets/76159594/2585f936-f02f-43d6-9c74-d7270d3bfa14)

brodokk commented:
The search bar for the emoji have also the same problem than the the language dropdown

![image](https://github.com/brodokk/Blueberries-Mastodon-UI-user-theme/assets/76159594/ae6044f7-9d41-4059-a5af-9271ffd01fa4)

brodokk commented:
Everything else seems good! 👍

HugoBDesigner committed:
Update blueberries.css

Fixed missing closing bracket in rule ".compose-form__poll-wrapper select option"

HugoBDesigner committed:
Update blueberries.css

Improved readability on emoji/language dropdown search bars.

HugoBDesigner commented:
Thanks for reviewing this PR! I fixed the missing bracket, reviewed the overall CSS for potential issues, and improved readability on search bars.

brodokk commented:
Sounds good to me! Thanks again for this PR!
